src/i18n/definitions.js

```diff
@@ -475,7 +478,6 @@ let strings = new LocalizedStrings(
       numberOfLearnedWordsMsg: "Du har lært {0} ord indtil videre.",
       correctOn: "Korrekt på: ",
 
-
       //Starred
       titleStarredWords: "Starred Words",
```

Comment:
I think it is a woops on my part, when doing the initial localization... - But shouldn't this be "Stjernemarkerede ord"?